examples/memstore-plugin/main.go

```diff
@@ -55,21 +53,11 @@ func main() {
 	opts := memory.Options{}
 	opts.InitFromViper(v)
 
-	traceCfg := &jaegerClientConfig.Configuration{
-		ServiceName: serviceName,
-		Sampler: &jaegerClientConfig.SamplerConfig{
-			Type:  "const",
-			Param: 1.0,
-		},
-		RPCMetrics: true,
-	}
-
-	tracer, closer, err := traceCfg.NewTracer()
+	tracer, err := jtracer.New("mem-store")
 	if err != nil {
-		panic("Failed to initialize tracer")
+		log.Fatal("Failed to initialize tracer", zap.Error(err))
```

Comment:
zap.Error() is from another logging library.
```suggestion
		panic(fmt.Errorf("Failed to initialize tracer: %w", err))
```